src/api/procedures/types.ts

```diff
@@ -600,6 +600,16 @@ export interface RemoveSecondaryAccountsParams {
   accounts: Account[];
 }
 
+export interface AccountWithSignature {
+  secondaryAccount: PermissionedAccount;
+  authSignature: string;
```

Comment:
How is someone supposed to get an `authSignature`? I think it needs more documentation